Multi-select filter (#147)
dimagi/open-chat-studio-docs: docs/changelog.md

# Changelog

## Aug 11, 2025

> * **CHANGE** Multi-select filters not have a "Select All" and "Clear All" option buttons located at the top of the dropdowns.

Reviewer: ```suggestion
* **CHANGE** Multi-select filters now have "Select All" and "Clear All" option buttons located at the top of the dropdowns.
```
